REPRODUCIBILITY.md

```diff
@@ -0,0 +1,72 @@
+# Reproducibility Guide
+
+In this guide, we explain how to regenerate the figures from our VLDB paper.
+Since the results depend on the hardware and toolchain, in order to get the exact same results, the experiments need to be run on the same machines.
+For the VLDB reproducibility process, we provide access to HPI computing infrastructure.
+Please reach out to us in order to set up your HPI account.
```

Comment:
Should we give email addresses here or is that clear from the CMT page? @lawben

Reply:
This should be fine. They have the paper, which has the email too